brix-core/src/main/java/org/brixcms/plugin/site/SitePlugin.java

```diff
@@ -649,6 +650,12 @@ public SiteTab(IModel<String> title, IModel<Workspace> workspaceModel) {
         public Panel newPanel(String panelId, IModel<Workspace> workspaceModel) {
             return new NodeManagerContainerPanel(panelId, workspaceModel);
         }
+        
+        @Override
+        public boolean isVisible() {
```

Comment:
Thank you Martin! Is this true even for implementation of org.apache.wicket.extensions.markup.html.tabs.ITab? Because it is not a component and has not setVisible method.